newton/_src/geometry/kernels.py

```
@@ -1266,6 +1266,240 @@ def create_soft_contacts(
+    if geo_type == GeoType.CONE:
+        d = sdf_cone(x_shape, geo_scale[0], geo_scale[1], int(Axis.Z))
+        n = sdf_cone_grad(x_shape, geo_scale[0], geo_scale[1], int(Axis.Z))
+
+    if geo_type == GeoType.ELLIPSOID:
+        d = sdf_ellipsoid(x_shape, geo_scale)
+        n = sdf_ellipsoid_grad(x_shape, geo_scale)
+
+    if geo_type == GeoType.MESH or geo_type == GeoType.CONVEX_MESH:
+        mesh = shape_source_ptr[rigid_shape]
+
+        face_index = int(0)
```

Ruff (0.16.3)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)